.github/workflows/issues-pinner.yml

```diff
@@ -0,0 +1,75 @@
+name: Issues pinner management
+
+on:
+  issues:
+    types:
+      - "pinned"
+      - "unpinned"
+
+
+permissions:
+  contents: none
+  issues: read
+
+# This allows a subsequently queued workflow run to interrupt/wait for previous runs
+concurrency:
+#  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.run_id }} :: ${{ github.event.action }}'
+#  cancel-in-progress: true
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.run_id }}'
+  cancel-in-progress: false
+
+jobs:
+
+  if-pinned:
```

Comment:
The code looks good to me, I would, however, remove the superfluous newlines below

Reply:
> I would, however, remove the superfluous newlines below

Do you refer to compact sourcecode?

Reply:
like
```
jobs:
  if-pinned:
    name: On pinning an issue
    if: github.event.action == 'pinned'
    permissions:
      issues: write
    runs-on: ubuntu-latest
    steps:
...
```

Reply:
> The code looks good to me, I would, however, remove the superfluous newlines below

Done. See commit cf3b28a730f4e56c7a48ee11af12e824c4aae4d8